src/main/java/thePackmaster/cards/quietpack/Weight.java

```diff
@@ -0,0 +1,48 @@
+package thePackmaster.cards.quietpack;
+
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import thePackmaster.cards.AbstractPackmasterCard;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+import static thePackmaster.util.Wiz.adp;
+import static thePackmaster.util.Wiz.atb;
+
+
+public class Weight extends AbstractPackmasterCard {
+    public final static String ID = makeID("Weight");
+
+    public Weight() {
+        super(ID, -2, CardType.STATUS, CardRarity.SPECIAL, CardTarget.NONE, CardColor.COLORLESS);
+        selfRetain = true;
+    }
+
+    @Override
+    public void triggerOnManualDiscard() {
+        AbstractCard c = this;
+        atb(new AbstractGameAction() {
+            @Override
+            public void update() {
+                adp().discardPile.moveToExhaustPile(c);
```

Comment:
```suggestion
                adp().discardPile.moveToExhaustPile(Weight.this);
```
You can access `this` but you need to add the class name before it so it knows which this you're referring to.